src/v/wasm/logger.h

```diff
@@ -16,4 +16,34 @@
 namespace wasm {
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
 extern ss::logger wasm_log;
+
+class logger {
+public:
+    struct meta {
+        ss::log_level level;
+        std::string_view transform_name;
+    };
```

Comment:
This is vestigal and can be removed right?

Reply:
Will be useful for plumbing into log_manager, but yeah meant to remove from this commit.

Reply:
💀